src/components/InputField/index.ts

```diff
@@ -0,0 +1,42 @@
+import Block from "../../utils/Block";
+import template from "./inputField.pug";
+import "./styles.css";
+interface InputFieldProps {
+  errorMsg?: string;
+  idInput?: string;
+  type?: string;
+  events?: {
+    click?: () => void;
+    focusin?: () => void;
+    focusout?: (env: Event) => void;
+  };
+  classes?: string;
+  inputClasses?: string;
+  valueInput?: string;
+  errorInput?: string;
+  placeholderInput?: string;
+  RegExp?: any;
+  RegInput?: any;
+  name?: string;
+  funBlur?: (env: FocusEvent) => void;
```

Comment:
Принято называть обработчики событий используя префикс on
```suggestion
  onBlur?: (env: FocusEvent) => void;
```